include/screen.h

```diff
@@ -360,8 +363,10 @@ class Screen : public GFXBase
 
         SSD1306Screen(int w, int h) : Screen(w, h)
         {
-            Heltec.begin(true /*DisplayEnable Enable*/, true /*LoRa Disable*/, false /*Serial Enable*/);
-            Heltec.display->screenRotate(ANGLE_180_DEGREE);
+            Heltec.begin(true /*DisplayEnable Enable*/, false /*LoRa Enable*/, false /*Serial Enable*/);
+            #ifdef SCREEN_ROTATION
+                Heltec.display->screenRotate(SCREEN_ROTATION);
+            #endif

```

Comment:
Why this change?

Reply:
On the LoRa version if you have it in the plastic case/housing thing from heltec the display is upside down and I figure might as well add a way to easily define the screens rotation.

Reply:
You mean that the original code turned the screen upside down on the LoRa version? (Which is indeed what rotating by 180 degrees should do, I guess...)

But in that case I think we should retain the old behavior for the non-LoRa version - the rotation was added to that for a reason. That is to say that for the non-LoRa version, `SCREEN_ROTATION` should then be defined to be `ANGLE_180_DEGREE`, maybe best within the definition block for the DEMO project in globals.h.

Reply:
Actually it should be defined in the 'platformio.ini' for any devices or environments that needs it. I do not know which devices need which rotations or why the code originally specified rotating the display 180 degrees. I suspect this was personal preference of whoever originally wrote that section of code. On the non Lora version it doesn't really matter which way the screen is oriented as far as I know.

Reply:
My point is that we should not make the existing projects/environments act differently while adding another. In the current heltecv3demo project the screen is rotated 180 degrees, so we should keep it that way even if we do not want to do the same for the LoRa version. 

Concerning the location of the define, I understand your statement, but this taps into a long-running conversation about what build config goes where. I'll make a decision about this myself and implement the define accordingly in this PR when I also consider the location of the `USE_SSD1306` one. (There's actually another aspect, which is that we prefer to not use `#ifdef` conditionals. I'll also include that in my considerations.)